core/drivers/crypto/caam/crypto.mk

```diff
@@ -0,0 +1,16 @@
+ifeq ($(CFG_NXP_CAAM),y)
+# Enable the crypto driver
+CFG_CRYPTO_DRIVER ?= y
+
+# Crypto Driver Debug
+# DRV_DBG_TRACE BIT32(0) // Driver trace
+# DRV_DBG_BUF   BIT32(1) // Driver dump Buffer
+CFG_CRYPTO_DRIVER_DEBUG ?= 0
+
+ifeq ($(CFG_LS),y)
+include core/drivers/crypto/caam/crypto_ls.mk
```

Comment:
Why not have the content of these two files directly in this file instead? There's quite a bit in common...

Reply:
indeed, thanks

Reply:
These files are still there. Moving everything into this file `core/drivers/crypto/caam/crypto.mk` would in my mind give a better overview and understanding.